code/modules/mob/mob_helpers.dm

```diff
@@ -29,6 +29,12 @@
 			return 1
 	return 0
 
+/proc/check_holy()

```

Comment:
This should be an atom proc or take an atom argument, not be an unattached proc.
I'm honestly surprised this was allowed to compile with the use of src.